pkg/acs/calacs/lib/prinfo.c

```diff
@@ -292,15 +292,15 @@ int maxch     i: allocated length of outstr (including EOS)
    to lower case, and pad with blanks.
 */
 
-void Upper1 (char *instr, char *outstr, int maxch) {
+void Upper1 (const char *instr, char *outstr, int maxch) {
 
 /* arguments:
 char *instr   i: input string
 char *outstr  o: string with only the first letter capitalized
 int maxch     i: allocated length of outstr (including EOS)
 */
 
-	int i;
+	int i = 1;
```

Comment:
When you set `'\0'` as first element, it means a null string, right? There is no need to pad a null string, as there is nothing to pad. Or did I misunderstand the code? We can always consult @stsci-hack as he wrote the original CALACS.